<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAlteraSenha</class>
 <widget class="QWidget" name="FormAlteraSenha">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Altera Senha</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 202, 206);
border-radius:5px</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>9</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Usuário</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/icon_title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>27</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>89</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 202, 206);
border-radius:5px</string>
   </property>
   <widget class="QLineEdit" name="txt_conf_senha">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>67</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>49</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirme a Senha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>9</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alt_senha">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>97</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:5px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:5px
}
</string>
    </property>
    <property name="text">
     <string>Alterar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/icons8-add-25.png</normaloff>icones/icons8-add-25.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_senha">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>27</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alt_senha_sair">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>97</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:5px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:5px
}
</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/icons-sinal-de-saída-25.png</normaloff>icones/icons-sinal-de-saída-25.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>txt_senha</tabstop>
  <tabstop>txt_conf_senha</tabstop>
  <tabstop>btn_alt_senha</tabstop>
  <tabstop>btn_alt_senha_sair</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_alt_senha_sair</sender>
   <signal>clicked()</signal>
   <receiver>FormAlteraSenha</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
